src/modules/coreHttpApi/openapi.yml

```diff
@@ -1909,16 +1909,9 @@ paths:
           application/json:
```

Comment:
The description a couple of lines of above still mentions the secret key

Reply:
Good catch, I only looked for `secretKey`